src/main/java/com/datastax/oss/driver/internal/core/loadbalancing/LatencyAndInflightCountLoadBalancingPolicy.java

```diff
@@ -0,0 +1,289 @@
+/*
```

Comment:
@SiyaoIsHiding mentioned this in separate conversation but I'm making a note of it here so that I don't forget it: the implementation below is nearly identical to the `LatencySensitiveLoadBalancingPolicy` in Jane's [earlier PR](https://github.com/apache/cassandra-java-driver/pull/1915) for this work.